riscv/trace_encoder_n.cc

```diff
@@ -0,0 +1,201 @@
+#include "trace_encoder_n.h"
+
```

Comment:
I noticed this file is a mix of 2-space and 4-space indentation. Can you standardize it all to 2-space, matching the rest of this repo?